src/main/kotlin/Server.kt

```diff
@@ -37,7 +39,11 @@ class Server(private val serverSocket: ServerSocket,
 
     fun acceptConnection() {
         socket = serverSocket.accept()
-        reader = BufferedReader(InputStreamReader(socket.getInputStream()))
+        val inputStream = socket.getInputStream()
+        while (socket.getInputStream().available() == 0) {
+            continue
+        }
```

Comment:
I think this is the same change as the one in PR #14.

Reply:
Yes! I couldn't add the `@structured-data` tag to the `AcceptanceTests` job in the `config.yml` file because it would run tests for all the features. In order to run the acceptance test for just the HTML feature. I had to add another command to the job. However, I found that when running circleci locally, I kept running into the `error reading from socket: Connection reset by peer`. Same issue as mentioned in PR #14. So I implemented this quick fix to get the acceptance tests to work.